protoc_plugin/test/generated_message_test.dart

```diff
@@ -807,12 +720,6 @@ void main() {
     message.m3M = p3.M_M();
   });
 
-  test('testToplevel', () {
-    var message = t.M();
-    message.t = T();
-    t.SApi(null);
```

Comment:
The call to `SApi` is not valid with null. Not sure what else to do other than delete this. 🤷

Reply:
Test moved to legacy test file.